src/main/Port.Adapter/UI/Views/Blazor/Shared/EditorBox.razor

```diff
@@ -59,6 +59,31 @@
                                     <ConditionalDisplay Show="@true">
                                         <li style="list-style:none" @key="@itemContext.Id">
                                             <input type="radio" id="@itemContext.Id" checked="@(this.EditTerminal != null && this.EditTerminal.Id == itemContext.Id)" @onchange="@(() => { this.EditTerminal = itemContext; })" />
+                                            @if (itemContext.Neuron.Type != Library.Common.RelativeType.NotSet)
```

Comment:
1. Are you able to move the logic below into the method GetAttribute? (See RenderFragment line 125 of BlazoriseSidebar.razor)
2. Then rename GetAttribute to GetSynapticSymbolSpan(itemContext.Neuron), and then evaluate strength, effect, and type in the method itself, then return the appropriate span RenderFragment.

Reply:
moved the logic and renamed the method